### PR TITLE
Signal Button Resprite

### LINKED ISSUE #18704: signal button needs resprite
it has perspective so it looks bad when rotated

### PR DESCRIPTION
**Pull request by ThunderBear2006**
## About the PR

The old sprites had depth and a shadow which looked weird when rotated, also the new sprites no longer sit at the center of a tile and are offset to appear to be actually *on* the wall they're placed on

Resolves #18704

**Media**

ohbe

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: The signal button now has new sprites

### CODE REVIEW

File: Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "x": 32,
     "y": 32
   },
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from CEV-Eris at commit 15748b71a1e6d2c547588c653d7c42f18011f7c5",
+  "license": "CC0-1.0",
```
Comment (lzk228) [inline Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json:7]: wha

Comment (ThunderBear2006) [reply Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json:7]: Could you elaborate?

Comment (metalgearsloth) [reply Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json:7]: anyway, these kinda look like derivatives of the old one so might be best to preserve the licensing and copyright.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json (hunk 1 above)
Comment (ThunderBear2006) [reply Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json]: > anyway, these kinda look like derivatives of the old one so might be best to preserve the licensing and copyright.

Yeah you're completely right, fixed my mistake

File: Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "y": 32
   },
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from CEV-Eris at commit 15748b71a1e6d2c547588c653d7c42f18011f7c5",
+  "copyright": "Derivitive of work from CEV-Eris at commit 15748b71a1e6d2c547588c653d7c42f18011f7c5",
```
Comment (EmoGarbage404) [inline Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json:8]:
```suggestion
  "copyright": "Taken from CEV-Eris at commit 15748b71a1e6d2c547588c653d7c42f18011f7c5, modified by ThunderBear2006 (github) for Space Station 14.",
```

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**